src/full_spectrum/neo4clj/internal/neo4j.clj

```diff
@@ -0,0 +1,167 @@
+(ns full-spectrum.neo4clj.internal.neo4j
```

Comment:
My personal preference is to drop `.internal` from the namespace(s).

It doesn't make the anymore internal having that name. Just don't mention them in documentation. Something that IMO would add more value is to write what your intention with an "internal" namespace is in the `ns` documentation:

```clojure
(ns neo4clj.neo4j
  "This namespace is intended for internal use of the library and is subject to
change. Don't expect function compatibility across version updates."
```

Also maybe the name of the NS could be improved. `...neo4j` doesn't give much away about what to expect to find in there :-)

The first that catches my attention when scanning through the NS is that several of the functions are converting so maybe it would make sense to split at least those into their own "convert/transform/whatnot" namespace.